src/frontend/contexts/ActiveProjectContext.tsx

```diff
@@ -46,23 +42,13 @@ export const ActiveProjectProvider = ({
 
         if (chosenActiveProject) {
           setActiveProjectId(chosenActiveProject.projectId);
-        } else {
-          createProject(undefined, {
-            onError: err => {
-              // TODO: Surface error in UI
-              console.error(err);
-            },
-            onSuccess: projectId => {
-              setActiveProjectId(projectId);
-            },
-          });
         }
       })
       .catch(err => {
         // TODO: Surface error in UI
         console.error(err);
       });
-  }, [activeProjectId, setActiveProjectId, createProject, mapeoApi]);
+  }, [activeProjectId, setActiveProjectId, mapeoApi]);
 
   if (!activeProjectId) {
     return <Loading />;
```

Comment:
this context shouldn't do this check anymore. We should do this check the level about (and that check is how we should determine whether to show the project onboarding screens).

I think `activeProjectId` should be a prop of this component